packages/web-components/.storybook/manager-head.html

```diff
@@ -14,7 +14,10 @@
 <meta property="og:title" content="Carbon for IBM.com Web Components">
 <meta property="og:site_name" content="Carbon for IBM.com Web Components">
 <meta property="og:description" content="This is the Web Components (custom elements) implementation of Carbon for IBM.com.">
-
+<meta
+  property="og:image"
+  content="https://media.github.ibm.com/user/52210/files/52a0cb80-8256-11eb-9bda-0e7a95354168"
```

Comment:
Let me know if you want me to upload to a place a bit more permanent (e.g. akamai server)

Reply:
Oo yes that would be nice! How do i do that lol

Reply:
James said there's a place for all placeholder images. I'd also be interested in updating those in the future. So Might be worth for me to figure it out